k8s/cert-manager/certificate.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: reconpoint-cert
+  namespace: default
+spec:
+  secretName: reconpoint-tls
+  issuerRef:
+    name: letsencrypt-prod
+    kind: ClusterIssuer
+  dnsNames:
+  - reconpoint.example.com
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)